entrypoint/upgradecluster/upgrademanual_test.go

```diff
@@ -33,35 +33,35 @@ var _ = Describe("Test:", func() {
 		)
 	})
 
-	It("Verifies ClusterIP Service", func() {
-		testcase.TestServiceClusterIp(false)
-	})
-
-	It("Verifies NodePort Service", func() {
-		testcase.TestServiceNodePort(false)
-	})
-
-	It("Verifies Ingress", func() {
-		testcase.TestIngress(false)
-	})
+	// It("Verifies ClusterIP Service", func() {
```

Comment:
This should not be commented out, right?

Reply:
tk you ,nope